students/Glogowska_Joanna/lesson_02_flow_control/chocolate_bar.py

```diff
@@ -0,0 +1,9 @@
+barlength = int(input('Height in cubes: '))
+barwidth = int(input('Width in cubes: '))
+n_squares = int(input('Is it possible to split it so that one of \
+the parts will have exactly this number of squares: '))
+if n_squares < barlength * barwidth and ((n_squares % barlength == 0) or
```

Comment:
You don't need the parentheses around `n_squares % barlength == 0` and this formatting is rather confusing, rather go for:
```
if ...
and (... or ...):
```